stock_quant_merge/models/stock.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# © 2015 OdooMRP team
+# © 2015 AvanzOSC
+# © 2015 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api
+
+
+class StockQuant(models.Model):
+    _inherit = 'stock.quant'
+
+    @api.multi
+    def merge_stock_quants(self):
+        pending_quants = self.filtered(lambda x: True)
```

Comment:
Wait, *you* wrote that didn't you? :-)
I think it avoids trying to merge a quant we just removed in a previous iteration.